web/turkey/survey/static/survey/js/auditors/visibility_changes.js

```diff
@@ -0,0 +1,17 @@
+// helper script file for focus_change auditors
```

Comment:
So biggest thing for me is if we do have multiple auditors depending on the same script file, what is the most proper way for them to load it, and does order matter?